Core/src/org/sleuthkit/autopsy/persona/OpenPersonasAction.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2014-2020 Basis Technology Corp.
```

Comment:
I don't think we need "2014-" here. since the file is created in 2020, just that should suffice.